src/components/Toaster/Toast/Toast.scss

```diff
@@ -49,24 +54,10 @@ $block: '.#{variables.$ns}toast';
             get-trailing-keyframe(toast-display-end, $indents-duration + $move-duration);
     }
 
-    &_show-animation#{&}_default {
-        $indents-duration: 0.35s;
-        $move-duration: 0.25s;
-
-        animation: toast-set-symmetrical-indents $indents-duration ease forwards,
-            toast-move-left $move-duration ease forwards $indents-duration,
-            get-trailing-keyframe(toast-display-end, $indents-duration + $move-duration);
-    }
```

Comment:
Why is animation changed in PR about customizing icon?

Reply:
In the previous version, the size of the icon was taken into account in the paddings. Because of this, it was necessary to handle the default toast type separately using `toast-set-symmetrical-indents`  keyframes. Now paddings always symmetrical.
I just remove duplicate animation rule.